app/controllers/planning_applications_controller.rb

```diff
@@ -16,7 +16,9 @@ def index
                              end
   end
 
-  def show; end
+  def show
+    @audit = @planning_application.audits.order("created_at ASC").last
```

Comment:
I think it'd be nicer to have the ordering as the default scope on the association, in the model